app/src/main/java/com/zulip/android/database/DatabaseHelper.java

```diff
@@ -30,7 +30,7 @@
     private static final String DATABASE_NAME = "zulip-%s.db";
     // any time you make changes to your database objects, you may have to
     // increase the database version
-    private static final int DATABASE_VERSION = 8;
+    private static final int DATABASE_VERSION = 9;
```

Comment:
@saketkumar95 we need to update the database version whenever we make changes to the existing database model so that the existing tables are dropped and new tables with updated schema are created.